hack/configure_local_webhook.sh

```diff
@@ -142,3 +142,11 @@ webhooks:
 EOF_CAT
 
 oc apply -n openstack -f ${TMPDIR}/patch_webhook_configurations.yaml
+
+# Scale-down operator deployment replicas to zero and remove OLM webhooks
+CSV_NAME="$(oc get csv -n openstack-operators -l operators.coreos.com/glance-operator.openstack-operators -o name)"
+
+if [ -n "${CSV_NAME}" ]; then
+  oc patch "${CSV_NAME}" -n openstack-operators --type=json -p="[{'op': 'replace', 'path': '/spec/install/spec/deployments/0/spec/replicas', 'value': 0}]"
+  oc patch "${CSV_NAME}" -n openstack-operators --type=json -p="[{'op': 'replace', 'path': '/spec/webhookdefinitions', 'value': []}]"
```

Comment:
As a follow up, but just out of curiosity, do we want to update [1] to reflect this approach? (`replace` w/ an empty array instead of `remove`).

[1] https://github.com/openstack-k8s-operators/dev-docs/blob/main/webhooks.md#disabling-webhooks

Reply:
Yes.  In fact, I had consulted that doc while creating this and realized that this was a better way to do it. :smile: